### PR TITLE
Hotels: Rename hotelFacilities to hotelAmenities

### PR DESCRIPTION
**Pull request by tbergq**
The button is called Amenities, this is to make naming more consistent

### CODE REVIEW

File: app/hotels/src/allHotels/NewAllHotelsSearch.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ class NewAllHotelsSearch extends React.Component<Props> {
       roomsConfiguration,
       currency,
       orderBy,
-      filterParams,
+      filterParams: { hotelAmenities, ...filterParmsRest },
```
Comment (RobinCsl) [inline app/hotels/src/allHotels/NewAllHotelsSearch.js]: `filterParmsRest`, it looks like there's a type in the variable name.

Comment (tbergq) [reply app/hotels/src/allHotels/NewAllHotelsSearch.js]: If you mean typo, you are correct, I will fix it

**Comment by tbergq** [comment]
@RobinCsl 🚀

File: app/hotels/src/allHotels/NewAllHotelsSearch.js (hunk 1 above)
Comment (RobinCsl) [reply app/hotels/src/allHotels/NewAllHotelsSearch.js]: Yes, I meant typo 😃

**Pull request merged**